storage/client/src/test/java/com/azure/storage/TestHelpers.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage;
+
+import com.azure.core.http.rest.Response;
+import com.azure.core.implementation.util.ImplUtils;
+import com.azure.storage.file.models.FileRef;
+import com.azure.storage.queue.models.CorsRule;
+import com.azure.storage.queue.models.Logging;
+import com.azure.storage.queue.models.Metrics;
+import com.azure.storage.queue.models.QueueItem;
+import com.azure.storage.queue.models.RetentionPolicy;
+import com.azure.storage.queue.models.SignedIdentifier;
+import com.azure.storage.queue.models.StorageErrorException;
+import com.azure.storage.queue.models.StorageServiceProperties;
+
+import java.time.Duration;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Contains helper methods for unit tests.
+ */
+public class TestHelpers {
```

Comment:
Should add an issue to centralize and de-duplicate functionality we added in the TestHelper class. Make two issues:

- Should centralize storage specific test helper methods into a Storage Common once it is created
- Should review test helper method across our SDKs and add them into TestBase